src/enums.ts

```diff
@@ -24,21 +24,18 @@
  *  THE SOFTWARE.
  */
 
-namespace powerbi.extensibility.visual {
+export enum DualKpiChartPositionType {
+    top = <any>"top",
```

Comment:
remove any from enums please. It's no longer requires since we updated TS version

Reply:
Done!